EWL 4166 add ipe stylesheet

chelsiejohnston opened the pull request:
_**BEFORE YOU REVIEW THIS, MERGE https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/28**_

## Ticket(s)

**Jira Ticket**

- [EWL-4166: Migrate IPE](https://issues.ama-assn.org/browse/EWL-4166)


## Description

We're not actually including the IPE Actions molecule from SG1 in SG2 for a few different reasons (we can't alter the markup anyway, there's no sense in showing it as a pattern) - but we still need to consume styling overrides for it from SG2 in d8.

- Adds a SCSS file with styling to supplement the default styles provided by the Panels IPE contrib module.


## To Test

- [ ] Follow the steps in https://github.com/AmericanMedicalAssociation/ama-d8/pull/195 to connect your local SG2/D8 environments for testing (so you can see the styles from this SG2 PR in d8 - preferably, use `develop` for the d8 repo) 
- [ ] `scripts/migrate`, then log in and edit http://ama-d8.local/test/topic_1
- [ ] Click "Edit" at the bottom of the screen
- [ ] See that the IPE Actions appear and that they look clean, useable, and visually similar to those in the current d8 theme. 
- [ ] In one of the IPE Actions blocks, click the pencil (edit) icon to edit the block. Verify that the form that displays is clean, usable, and visually similar to what's in the current d8 theme.

## Visual Regressions

N/A. Visual regression testing in SG2/D8 is dependent upon https://issues.ama-assn.org/browse/EWL-3503.


## Relevant Screenshots/GIFs

![screen shot 2017-12-26 at 5 01 00 pm](https://user-images.githubusercontent.com/12160398/34366208-5f6031aa-ea5e-11e7-8803-438a89d1840c.png)

![screen shot 2017-12-26 at 4 59 39 pm](https://user-images.githubusercontent.com/12160398/34366204-54aaf9b6-ea5e-11e7-860d-d8810c9cf4ba.png)


## Remaining Tasks

- We may want to revisit this after more of the Topic page is built out in SG2/the new theme.


## Additional Notes

N/A

csj1012 committed:
[EWL-4166] added empty file

csj1012 committed:
[EWL-4166] pasted in the old scss

csj1012 committed:
[EWL-4166] add some cleanup styling

designerneil reviewed (changes requested):
Let's move this stylesheet out of our patterns directory and into it's own directory. Perhaps called misc or ipe

csj1012 committed:
[EWL-4185] fix the variant links.

csj1012 committed:
[EWL-4185] fix the variant links.

csj1012 committed:
[EWL-4185] fix the variant links.

designerneil committed:
gulpfile task to watch when twig files change and exports them public/twig

Commit:
refs EWL-4420: add class option to the paragraph data model

designerneil committed:
Updates per PR requests

designerneil committed:
Finishes social icons organism

designerneil committed:
Adds markdown file

designerneil committed:
removes date block

csj1012 committed:
[EWL-4214] add newlines, Membership With CTA .md file, formatting

csj1012 committed:
[EWL-4214] removed Topic Promotion patterns/assets from this PR (they're included in a separate PR)

csj1012 committed:
[EWL-4214] renamed the pattern from "Topic Membership" to just "Membership"

Commit:
refs EWL-4420: add json, md, and twig files for the inline bio

Commit:
refs EWL-4420: add json, md, and twig files for the inline bio

Commit:
refs EWL-4420: fix the paragraphs showing correctly

Commit:
refs EWL-4420: fix the paragraphs showing correctly

Commit:
refs EWL-4420: rename reset so that it's being pulled in

Commit:
refs EWL-4420: rename reset so that it's being pulled in

designerneil committed:
Updates typography and adds new typography from UX

designerneil committed:
Updates typography and adds new typography from UX

designerneil committed:
adds line break

designerneil committed:
adds line break

Commit:
refs EWL-4420: inital inline bio styles

Commit:
refs EWL-4420: inital inline bio styles

csj1012 committed:
newlines

Commit:
put css tasks back in gulpfile.

Commit:
move reset to be picked up by gulp and add link to reset in head.twig

csj1012 committed:
[EWL-4423] add the pattern files and styling

csj1012 committed:
[EWL-4423] change tag atom to use the link atom

Commit:
Merge branch 'bug/fix-css-reset' into feature/EWL-4420-inline-bio

csj1012 committed:
[EWL-4423] add pattern info files

designerneil committed:
Changes grey to gray for color variables

designerneil committed:
adds line break

designerneil committed:
Adds inline block copy molecule
Updates button to accommodate secondary styling

designerneil committed:
Adds inline block copy molecule
Updates button to accommodate secondary styling

Commit:
refs EWL-4420: update styling for inline bio

Commit:
refs EWL-4420: update styling for inline bio

Commit:
fix lack of BEM syntax for link classes

Commit:
fix lack of BEM syntax for link classes

Commit:
Merge pull request #46 from AmericanMedicalAssociation/bug/change-grey-to-gray

Changes grey to gray for color variables

Commit:
Merge branch 'develop' into feature/EWL-4420-inline-bio

Commit:
refs EWL-4420: fix change in gray v. grey

Commit:
refs EWL-4420: fix change in gray v. grey

Commit:
Merge pull request #38 from AmericanMedicalAssociation/feature/EWL-4214-migrate-membership

EWL-4214 SG2 | Migrate Membership organism

Commit:
Merge pull request #37 from AmericanMedicalAssociation/feature/EWL-4162-topic-page

Feature/EWL-4192 Article Preview - Help D8 render  a 3 column layout

Commit:
remove css task from the gulpfile

Commit:
Merge branch 'develop' into bug/fix-css-reset

designerneil committed:
udpates font size per wireframe mockup

designerneil committed:
udpates font size per wireframe mockup

designerneil committed:
udpates font size per wireframe mockup

designerneil committed:
udpates font size per wireframe mockup

Commit:
Merge branch 'develop' into bug/fix-css-reset

designerneil committed:
Updates links to match font sizes from mockup

designerneil committed:
Updates links to match font sizes from mockup

Commit:
ensure reset scss is working and resolve broken variable names

csj1012 committed:
[EWL-4432] add files

csj1012 committed:
[EWL-4422] update the font sizes

Commit:
refs EWL-4420: cleanup twig

Commit:
refs EWL-4420: cleanup twig

designerneil committed:
Merge pull request #44 from AmericanMedicalAssociation/bug/fix-css-reset

Fix css reset

Commit:
refs EWL-4420: create a variant of link to include icon

Commit:
refs EWL-4420: create a variant of link to include icon

Commit:
refs EWL-4420: update link.md with documentation about variant

Commit:
refs EWL-4420: update link.md with documentation about variant

designerneil committed:
Merge pull request #40 from AmericanMedicalAssociation/feature/EWL-4422-prototype-inline-article

[EWL-4422] Prototype- "Article Preview as Inline" pattern

designerneil committed:
removes top level json

designerneil committed:
removes top level json

Commit:
refs EWL-4420: add twitter link to inline bio

Commit:
refs EWL-4420: add twitter link to inline bio

Commit:
Merge branch 'develop' into feature/EWL-4420-inline-bio

designerneil committed:
renames ama__type-smaller to ama__type-small so it compiles

designerneil committed:
renames ama__type-smaller to ama__type-small so it compiles

designerneil committed:
Merge branch 'feature/EWL-4429-inlne-news-article' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4429-inlne-news-article

Commit:
Merge pull request #35 from AmericanMedicalAssociation/bugfix/add-template-markdown

Adds topic template markdown files

Commit:
Merge pull request #42 from AmericanMedicalAssociation/feature/EWL-4419-date-block

EWL-4419 SG2 | Create Date atom

Commit:
Merge remote-tracking branch 'origin/feature/EWL-4420-inline-bio' into feature/EWL-4421-article-header

Commit:
Merge remote-tracking branch 'origin/feature/EWL-4418-create-social-share' into feature/EWL-4421-article-header

csj1012 committed:
[EWL-4432] remove top level from article-preview.json

Commit:
refs EWL-4421: initial article header info

Commit:
refs EWL-4421: clean up other patterns to include

Commit:
refs EWL-4420: update inline bio md to be correct

Commit:
refs EWL-4420: update inline bio md to be correct

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

designerneil committed:
Start of news page

csj1012 committed:
[EWL-4432] rename Topic Article Preview to Article Stub

designerneil committed:
Merge branch 'feature/EWL-4418-create-social-share' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4418-create-social-share

# Conflicts:
#	styleguide/gulpfile.js
#	styleguide/package.json
#	styleguide/source/_meta/_00-head.twig

designerneil committed:
Adds empty lines

Commit:
Merge branch 'feature/EWL-4418-create-social-share' into feature/EWL-4421-article-header

designerneil committed:
Adds empty lines

designerneil committed:
Merge pull request #50 from AmericanMedicalAssociation/feature/EWL-4432-rename-articles

[EWL-4432] rename "Topic Article Preview" to "Article Stub"

designerneil committed:
Merge branch 'feature/EWL-4424-create-news-article-template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4424-create-news-article-template

designerneil committed:
Fixes class per PR request

designerneil committed:
Fixes class per PR request

designerneil committed:
Adds empty lines

designerneil committed:
Adds empty lines

designerneil committed:
Merge branch 'feature/EWL-4420-inline-bio' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4420-inline-bio

designerneil committed:
Merge pull request #48 from AmericanMedicalAssociation/feature/EWL-4420-inline-bio

EWL-4420 Inline Bio Molecule

designerneil committed:
Merge branch 'feature/EWL-4424-create-news-article-template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4424-create-news-article-template

designerneil committed:
Adds paragraph news article

designerneil committed:
Adds paragraph news article

designerneil committed:
Adds paragraph news article

designerneil committed:
Adds paragraph news article

designerneil committed:
Adds paragraph news article

designerneil committed:
Adds paragraph news article

designerneil committed:
Adds paragraph news article

csj1012 committed:
[EWL-4423] add newlines and replace $grey-7

Commit:
Merge pull request #41 from AmericanMedicalAssociation/feature/EWL-4418-create-social-share

EWL-4418 | Create social share molecule

Commit:
Merge pull request #47 from AmericanMedicalAssociation/feature/EWL-4429-inlne-news-article

EWL-4429 Adds inline block copy molecule

Commit:
Merge branch 'develop' into feature/EWL-4421-article-header

Commit:
display article header

designerneil committed:
Merge branch 'feature/EWL-4434-update-typography' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4434-update-typography

csj1012 committed:
Merge branch 'develop' into feature/EWL-4423-prototype-tags

# Conflicts:
#	styleguide/source/assets/scss/00-base/_colors.scss

designerneil committed:
Adds date block

csj1012 committed:
[EWL-4434] fix nesting issue

csj1012 committed:
Merge pull request #43 from AmericanMedicalAssociation/feature/EWL-4434-update-typography

EWL-4434 Updates typography and adds new typography from UX

csj1012 committed:
Merge branch 'develop' into feature/EWL-4432-prototype-four-up-teaser

Commit:
Merge pull request #52 from AmericanMedicalAssociation/feature/EWL-4419-date-block

EWL-4419 SG2 | Create Date atom

csj1012 committed:
Merge branch 'feature/EWL-4432-rename-articles' into feature/EWL-4432-prototype-four-up-teaser

Commit:
Merge branch 'develop' into feature/EWL-4421-article-header

designerneil committed:
Removes extraneous topic promo markdown file

designerneil committed:
Removes extraneous topic promo markdown file

designerneil committed:
Merge branch 'feature/EWL-4185-topic-promo' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4185-topic-promo

designerneil committed:
updates markdown

designerneil committed:
updates markdown

csj1012 committed:
[EWL-4432] add .json to metadata

csj1012 committed:
[EWL-4432] make stub pattern pass data to metadata pattern

csj1012 committed:
[EWL-4432] get all the correct data to show up

csj1012 committed:
[EWL-4434] add the element layout for the pattern

Commit:
refs EWL-4421: fix float overlap with share and add back image

designerneil committed:
restores topic promo
renames topic promo to just promo

designerneil committed:
restores topic promo
renames topic promo to just promo

csj1012 committed:
Merge branch 'feature/EWL-4432-prototype-four-up-teaser' into feature/EWL-4433-prototype-column-teaser

csj1012 committed:
[EWL-4433] add the pattern files

Commit:
Merge pull request #45 from AmericanMedicalAssociation/feature/EWL-4423-prototype-tags

EWL-4423 SG2 | Create tag atom and tags molecule

csj1012 committed:
[EWL-4432] newlines and syntax cleanups

csj1012 committed:
[EWL-4432] newlines

Commit:
Merge pull request #53 from AmericanMedicalAssociation/feature/EWL-4432-prototype-four-up-teaser

Feature/ewl 4432 prototype four up teaser

designerneil committed:
Adds tools block

designerneil committed:
Changes made per pr request

Commit:
Merge pull request #55 from AmericanMedicalAssociation/feature/EWL-4165-tools

EWL-4165 Migrate topic tools organism

designerneil committed:
Merge branch 'feature/EWL-4424-create-news-article-template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4424-create-news-article-template

designerneil committed:
Merge branch 'feature/EWL-4424-create-news-article-template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4424-create-news-article-template

csj1012 committed:
Merge branch 'develop' into feature/EWL-4433-prototype-column-teaser

csj1012 committed:
[EWL-4433] add newline and correct BEM naming for link class

Commit:
Merge pull request #54 from AmericanMedicalAssociation/feature/EWL-4433-prototype-column-teaser

Feature/ewl 4433 prototype column teaser

Commit:
remove clearfix and floats and fix typo

designerneil committed:
Updates the article preview as inline by adding a parent container to the json
Updates the four up teaser by adding a parent container to the json

designerneil committed:
Updates the article preview as inline by adding a parent container to the json
Updates the four up teaser by adding a parent container to the json

designerneil committed:
Updates the article preview as inline by adding a parent container to the json
Updates the four up teaser by adding a parent container to the json

designerneil committed:
Updates the article preview as inline by adding a parent container to the json
Updates the four up teaser by adding a parent container to the json

designerneil committed:
Address concerns made in PR

designerneil committed:
Address concerns made in PR

designerneil committed:
Merge branch 'feature/EWL-4185-topic-promo' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4185-topic-promo

Commit:
Merge pull request #36 from AmericanMedicalAssociation/feature/EWL-4185-topic-promo

EWL-4185 topic promotion organism

designerneil committed:
Adds h2 header to body

designerneil committed:
Adds h2 header to body

designerneil committed:
Adds h2 header to body

designerneil committed:
Adds h2 header to body

Commit:
Merge branch 'develop' into feature/EWL-4421-article-header

Commit:
remove share on mobile

csj1012 committed:
Merge pull request #49 from AmericanMedicalAssociation/feature/EWL-4421-article-header

EWL-4421 article header

designerneil committed:
Adds pattern teaser

designerneil committed:
Adds pattern teaser

designerneil committed:
Merge branch 'feature/EWL-4424-create-news-article-template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4424-create-news-article-template

designerneil committed:
Adds article header block
Adds inline block copy

designerneil committed:
Merge branch 'feature/EWL-4424-create-news-article-template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4424-create-news-article-template

csj1012 committed:
Merge pull request #51 from AmericanMedicalAssociation/feature/EWL-4424-create-news-article-template

EWL-4424 create news article template

csj1012 committed:
[EWL-4124] fixed a regression in the button/cta behavior for the pattern.

csj1012 committed:
[EWL-4124] fixed hover text color regression

Commit:
update package.json with latest release

Commit:
Merge pull request #56 from AmericanMedicalAssociation/feature/EWL-4124-membership-adjustments

[EWL-4124] SG2 | Migrate "Membership" pattern - regression fixes

csj1012 committed:
[EWL-4388] added pattern files

csj1012 committed:
[EWL-4388] added pattern files

csj1012 committed:
[EWL-4388] added pattern files

csj1012 committed:
[EWL-4388] add pattern to the Topic page pattern and start populating .json data

csj1012 committed:
[EWL-4388] add pattern to the Topic page pattern and start populating .json data

csj1012 committed:
[EWL-4388] add pattern to the Topic page pattern and start populating .json data

designerneil committed:
Updates topic preview styling and video sizes

designerneil committed:
removes article preview molecule
updates article stub molecule to match article preview
changes article stub to help D8 out when consuming it

designerneil committed:
updates classes to ama__article-stub__metadata

designerneil committed:
Updates made to article preview per PR https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/61

designerneil committed:
Adds article-stub related markdown

designerneil committed:
Adds article small variant to lead news

designerneil committed:
Adds article small variant to lead news

designerneil committed:
Merge branch 'feature/EWL-4388-migrate-lead-news-block' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4388-migrate-lead-news-block

designerneil committed:
Adds lead news markdown

designerneil committed:
Adds lead news markdown

Commit:
update naming for as variant

designerneil committed:
removes extraneous article stub as related markdown

designerneil committed:
renames article stub variants with as variant

Commit:
Merge pull request #61 from AmericanMedicalAssociation/feature/EWL-4192-Migrate-Article-Preview

EWL-4192 Article Preview

designerneil committed:
removes old colors and fonts
updates all files that used them

designerneil committed:
removes old colors and fonts
updates all files that used them

designerneil committed:
Adds new reds and greens

designerneil committed:
Adds new reds and greens

designerneil committed:
Merge branch 'feature/EWL-4388-migrate-lead-news-block' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4388-migrate-lead-news-block

designerneil committed:
Fixes lead news per PR
- Adds related content article stub
- modifies title only
- modifies metadata to use a parent container
- changes some of the conditionals for article stub

designerneil committed:
Fixes type sizes
take away unnecessary parent selector in json

designerneil committed:
removes paragraph from json

designerneil committed:
Fixes type size for teaser

designerneil committed:
Adds social share buttons

designerneil committed:
Adds social share buttons

designerneil committed:
Adds social share buttons

designerneil committed:
Adds social share buttons

designerneil committed:
changes date block to use h3

designerneil committed:
Adjust bio image to have a max of 80px

designerneil committed:
tags now have border on top and button
tags uses flexbox instead of inline elements
margin and padding have been adjusted to give the tags more air

designerneil committed:
Adds learn more to tools block

Commit:
Merge pull request #70 from AmericanMedicalAssociation/bugfix/EWL-4419-date-atom

EWL-4419 changes date atom to use h3

Commit:
fix empty line

Commit:
Merge pull request #73 from AmericanMedicalAssociation/bugfix/EWL-4430-tools-list-footer

EWL-4430 Adds learn more to tools block

Commit:
Merge pull request #72 from AmericanMedicalAssociation/bugfix/EWL-4423-tag-atom

EWL-4423 Adjust tags styling and spacing

Commit:
Merge pull request #71 from AmericanMedicalAssociation/bugfix/EWL-4420-inline-bio

EWL-4420 Adjust bio image to have a max of 80px

Commit:
Merge pull request #68 from AmericanMedicalAssociation/bugfix/EWL-4433-Column-teaser

EWL-4433 Fixes type size for the column teaser to be smaller

designerneil committed:
tools clean and integration with D8

designerneil committed:
updates per PR suggestion

designerneil committed:
Address concerns made in PR
removes unused font mixin file

designerneil committed:
Merge branch 'bugfix/EWL-4418-social-share' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4418-social-share

designerneil committed:
Adds empty line

designerneil committed:
Adds empty line

designerneil committed:
adds learn more tools footer

Commit:
Add empty line

Commit:
Merge pull request #74 from AmericanMedicalAssociation/feature/EWL-4165-tools

tools clean and integration with D8

Commit:
Add empty line

Commit:
Merge pull request #66 from AmericanMedicalAssociation/bugfix/EWL-4432-four-up-teaser

Fixes type sizes for Article Stub and Four Up Teaser

designerneil committed:
changes names of article-metadata to better reflect BEM standards for what the molecule is

designerneil committed:
Merge branch 'chore/EWL-4455-remove-old-typography-colors' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into chore/EWL-4455-remove-old-typography-colors

Commit:
Merge pull request #59 from AmericanMedicalAssociation/feature/EWL-4388-migrate-lead-news-block

[EWL-4388] Migrate Lead News Block

Commit:
Merge branch 'develop' into chore/EWL-4455-remove-old-typography-colors

Commit:
Merge pull request #64 from AmericanMedicalAssociation/chore/EWL-4455-remove-old-typography-colors

EWL-4455 removes old colors and fonts

designerneil committed:
Merge branch 'bugfix/EWL-4418-social-share' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4418-social-share

Commit:
Merge pull request #69 from AmericanMedicalAssociation/bugfix/EWL-4418-social-share

EWL-4418 Adds social share buttons to article header

designerneil committed:
Adds IPE editor molecule

designerneil committed:
Updates IPE editor by removing extraneous css

designerneil committed:
Removes calls for IPE css in header

designerneil committed:
Merge branch 'feature/EWL-4166-add-ipe-stylesheet' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4166-add-ipe-stylesheet
